src/Avalonia.Native/WindowImplBase.cs

```diff
@@ -509,6 +512,26 @@ public void SetFrameThemeVariant(PlatformThemeVariant themeVariant)
         }
 
         public AcrylicPlatformCompensationLevels AcrylicCompensationLevels { get; } = new AcrylicPlatformCompensationLevels(1, 0, 0);
+
+        public async Task<IDisposable> RequestPlatformInhibition(PlatformInhibitionType type, string reason)
```

Comment:
I think there is a misunderstanding.
TryGetFeature correctly returns _platformBehaviorInhibition, as it should.

But `Task<IDisposable> RequestPlatformInhibition` should be moved to the TopLevel control. As it's not platform specific. And stubs can be removed from other backends.

Reply:
I.e., when TryGetFeature returns correct IPlatformBehaviorInhibition only on macos backend (as for now), it will just return null on other backends, falling back to the `return new ExecuteActionDisposer` route in the TopLevel code (or `Disposable.Create`).